### PR TITLE
About us and Newsletter

### LINKED ISSUE #7: Newsletter & About Us - Stina

### PR DESCRIPTION
**Pull request by ssmeds**
Made Mobile, tablet and desktop as best as I could :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blue Recycling</title>
+    <link rel="stylesheet" href="styling.css">
+</head>
+
+<body>
+    <div class="wrapper-sections">
```
Comment (ljwedin) [inline index.html]: We decided on using camel case instead of kebab casing

File: index.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        <div class='section' id='newsletter'>
```
Comment (ljwedin) [inline index.html]: You could use the section tag instead of div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            <div class="tablet">
+                <span id='join'>Join our</span>
```
Comment (ljwedin) [inline index.html]: Maybe a bit generic id's? Could be something like joinSpan and newsSpan or something similar

File: index.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+                <span id='news'>newsletter</span>
+                <p id='desktop-text'>Want to get the best and latest news from us? Fill in your email adress and we will
+                    update you with the
+                    latest news and how we will continue to strive for a better climate.</p>
+                <form>
+                    <input type="email" name="" id="">
```
Comment (ljwedin) [inline index.html]: Do we need the name attribute of it's empty?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+        </div>
+        <div class='section' id='about-us'>
+            <div id="tablet-about-us">
+                <span id='about'>About</span> <span id='us'>us</span>
+            </div>
+            <p>We at Blue Recycle love the enviroment and fashion. By choosing sustainable clothes and recycling your
+                worn
+                denim, you can help make a positive impact on the planet. The more we recycle, the more good we can do.
+            </p>
+            <div id='svg-div'>
```
Comment (ljwedin) [inline index.html]: Also maybe a bit generic id with risk of being reused in other places. Could it be a class instead?

**Review by ljwedin — CHANGES_REQUESTED** [review]
Take a look at the case type and a glance at the other stuff, I've tested the code though and it looks great!

**Comment by smaristeinar** [comment]
looks good

**Review by smaristeinar — APPROVED** [review]

**Pull request merged**